fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedDocValuesFormat.java

```diff
@@ -42,12 +44,17 @@
  * data needed to be read from FDB.
  *
  */
+@AutoService(DocValuesFormat.class)
 public class LuceneOptimizedDocValuesFormat extends DocValuesFormat {
 
     private DocValuesFormat docValuesFormat;
 
+    public LuceneOptimizedDocValuesFormat() {
```

Comment:
No-one is calling this constructor - is it necessary?

Reply:
Yes, the SPILoader calls it...